example/aligns.js

```diff
@@ -17,16 +17,16 @@ const tablePlugin = PluginEditTable({
 /*
  * Set align data for the current column
  */
-function setColumnAlign(change: Change, align: string): Change {
-    const pos = tablePlugin.utils.getPosition(change.value);
-    const columnCells = tablePlugin.utils.getCellsAtColumn(
+function setColumnAlign(editor: Editor, align: string): Editor {
```

Comment:
WHAT IS THIS DARK MAGIC?!

Reply:
Flow innit - because some peeps want JS to be type safe. pah